### DISCUSSION TITLE
autofocus editor on page load on iOS browsers?

### DISCUSSION THREAD

**Discussion by jsjoeio** [ueberdosis/tiptap#5504] [General Questions]
Has anyone had any luck with this? I've tried both `editor.commands.focus("end")` and `editor.chain().focus().run();` in a React app but no luck. I saw this [StackOverflow post](https://stackoverflow.com/a/55425845/3015595) about how Nike does it for inputs but couldn't figure out which element to use it on for TipTap.

Any suggestions?

**Comment by GreedyWhale** [comment]
You can pass in `autofocus ` for autofocus at initialization time.

[autofocus](https://tiptap.dev/docs/editor/api/editor#autofocus)

It can also be done by calling the methods of the editor instance after initializing the editor

[focus](https://tiptap.dev/docs/editor/api/commands/selection/focus#page-title)
```
const editor = new Editor();

editor.commands.focus()
```

**Reply by jsjoeio** [reply]
does this work on iOS browsers?

**Reply by nperez0111** [reply]
It should, but sometimes React can re-render and the focus gets lost when that happens. If you get the rendering to run only once on mount it should work just fine. 

Do also note that in dev mode Strict mode will cause rerenders so it may not work in dev but work in prod

**Comment by DhaniAM** [comment]
You can call this dynamically or inside the editor `onCreate`:
```js
editor.commands.focus("start");

```
